jreleaser.yml

```diff
@@ -0,0 +1,58 @@
+environment:
+  properties:
+    jdkPathPrefix: 'target/jdks'
+    jdkFilePrefix: 'zulu17.32.13-ca-jdk17.0.2'
+    artifactsDir: out/jreleaser/assemble/liquibase/jpackage
+
+project:
+  name: liquibase
+  description: Main Liquibase Source
+  longDescription: Main Liquibase Source
+  links:
+    homepage: https://www.liquibase.org/
+  authors:
+    - Liquibase
+  license: APACHE-2.0
+  inceptionYear: 2023
+  stereotype: cli
+  java:
+    version: 11
+    groupId: org.liquibase
+    artifactId: liquibase
+    mainClass: liquibase.integration.commandline.LiquibaseLauncher
+
+assemble:
+  jlink:
+    liquibase-jink:
+      active: ALWAYS
+      exported: false
+      imageName: 'liquibase-{{projectVersion}}'
+      executable: liquibase
+      targetJdks:
+        - path: '{{jdkPathPrefix}}/zulu17LinuxIntel/{{jdkFilePrefix}}-linux_x64'
```

Comment:
Custom JDKs must be explicitly provisioned, JReleaser will not perform this task.
Have a look at https://github.com/jreleaser/helloworld-java-jpackage for more information.

Reply:
Thanks @aalmiray It was just a first test. Also since the `liquibase` final artifact is already generated in the build process (a `.tgz` file under https://github.com/liquibase/liquibase/tree/master/liquibase-dist/target) do you think it is possible to just use `jrealease` (packager) to create `deb`/`rpm` packages based on that `.tgz` file? I was tryting to find an example using `maven` so I can ommit using the `jreleaser.yml` file as well and just configure `jrealease` in the main `pom.xml`.

Reply:
It's definitely possible to use the `.tgz` but its contents must be explicitly expanded and mapped to the JReleaser configuration. Perhaps it would be a good idea to support archives (zips and tars) as sources for JReleaser's `jlink` assembler.

Reply:
Filed https://github.com/jreleaser/jreleaser/issues/1399

Reply:
I opened this as well https://github.com/jreleaser/jreleaser/issues/1400